sunpy/net/hek/tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+from regions import PointSkyRegion, PolygonSkyRegion
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.time import Time
+
+from sunpy.net.hek.utils import get_unit, parse_chaincode
+
+
+def test_get_unit():
+    erg_per_cm3 = u.def_unit("ergs/cm^3", u.erg/u.ml)
+
+    u1 = get_unit('DN/sec/pixel')
+    u2 = get_unit('ergs per cubic centimeter')
+    u3 = get_unit('m/s/s')
+
+    assert u1 == u.Unit('DN / (pix s)')
+    assert u2 == erg_per_cm3
+    assert u3 == u.Unit('m / s2')
+
+def test_parse_chaincode_helioprojective():
+    value = "POLYGON((10 20, 30 40, 50 60))"
+    attribute = {"frame": "helioprojective"}
+    unit = "arcsec"
+    time = Time('2011-08-09 12:04:00.000')
+
+    result = parse_chaincode(value, attribute, unit, time)
+
+    vertices = SkyCoord([10, 30, 50] * u.arcsec, [20, 40, 60] * u.arcsec, obstime=time, observer='earth', frame="helioprojective")
+    region = PolygonSkyRegion(vertices=vertices)
+
+    assert region == result
```

Comment:
What does the equality operator for `PolygonSkyRegion` even check?  I note that the next line explicitly checks the coordinate values using a `SkyCoord` method.